app/webpacker/components/DuplicateChecker/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useMutation, useQuery, useQueryClient } from '@tanstack/react-query';
+import WCAQueryClientProvider from '../../lib/providers/WCAQueryClientProvider';
+import Loading from '../Requests/Loading';
+import Errored from '../Requests/Errored';
+import computePotentialDuplicates from './api/computePotentialDuplicates';
+import getPotentialDuplicatesData from './api/getPotentialDuplicatesData';
+import SimilarPersons from './SimilarPersons';
+import DuplicateCheckerHeader from './DuplicateCheckerHeader';
+
+export default function Wrapper({ competitionId }) {
+  return (
+    <WCAQueryClientProvider>
+      <DuplicateChecker competitionId={competitionId} />
+    </WCAQueryClientProvider>
+  );
+}
+
+function DuplicateChecker({ competitionId }) {
+  const { data, isLoading, isError } = useQuery({
+    queryKey: ['potential-duplicates-data'],
+    queryFn: () => getPotentialDuplicatesData({ competitionId }),
```

Comment:
If `competitionId` is part of the `queryFn` parameters, it *must* be part of the `queryKey` as well.

Reply:
Oh yes, I missed it. Thanks for pointing. I've changed it now.